frontend/explore_herds.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * @file This file contains the ExploreHerds function. This function is used for
+ * selecting herds and displaying them with the Herds function.
+ */
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { Tabs, Tab } from '@material-ui/core/'
+import { Box, Button } from '@material-ui/core'
+import { Herd } from '~herd'
+import { Genebank } from 'data_context_global'
+
+import { useDataContext } from './data_context'
+
+// Define styles for tab menu
+const useStyles = makeStyles({
+  sidebar: {
+    float: "left",
```

Comment:
Some interesting overflow is happening here
![image](https://user-images.githubusercontent.com/199380/85532425-fa666280-b60f-11ea-8ab0-1ebdebb42ad1.png)